Style.css

```diff
@@ -0,0 +1,36 @@
+*body {
+  margin: 0;
+  padding: 0;
+}
+
+#navbar {
+  width: 300px;
+  position: fixed;
+  background-color: lightgray;
+  height: 100vh;
+  left: 0;
+  top: 0;
+  box-shadow: 5px 5px 5px gray;
+}
```

Comment:
Good use of inline comments. Well done. 

We try to give reasons why we suggest these changes so the author has an opportunity to learn from you. The reason may be obvious to you but not to the author of the code. 

- [ ] Kindly rephrase your comments and state the reasons for the suggestions.